src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -21,4 +21,7 @@
         <property name="java.util.logging.config.file" value="WEB-INF/logging.properties"/>
     </system-properties>
 
+    <automatic-scaling>
+        <min-idle-instances>1</min-idle-instances>
```

Comment:
It's our website anyway, we don't want to have cold starts. And if the traffic is low enough, it's still free (the first 28h are free)